spring-ai-vector-store/pom.xml

```diff
@@ -47,6 +47,17 @@
 			<version>${project.parent.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-tracing-bridge-otel</artifactId>
```

Comment:
why?

Reply:
This should have been merged from the upstream main branch before and has been removed.